sn_node/src/put_validation.rs

```diff
@@ -735,6 +741,15 @@ impl Node {
             }
         }
 
+        if parent_is_a_double_spend && all_verified_spends.len() == 1 {
+            warn!("Parent is a double spend for {unique_pubkey:?}, ignoring this spend");
+            return Err(Error::InvalidRequest(format!(
+                "Parent is a double spend for {unique_pubkey:?}, ignoring this spend."
+            )));
```

Comment:
Maybe `InvalidRequest` is not exactly the correct error type here?
We have `TransferError::InvalidParentSpend` would that be better?

Reply:
Yeah! That would be perfect.